src/services/game.service.ts

```diff
@@ -37,7 +40,53 @@ class GameService {
 
   private async wordWasUsedInGame(word: string, gameId: string): Promise<boolean> {
     const rounds = await roundService.getAllByGameId(gameId);
-    return rounds.some(round => round.words.includes(word));
+    return rounds.some(round => round.words.map(obj => obj.word).includes(word));
+  }
+
+  async start(id: string) {
+    await gameRepository.start(id);
+    const game = await this.getById(id);
+    const roundId = await this.createRound(game);
+    game.rounds.push(roundId);
+    await this.update(game);
+    //TODO provide chat
+  }
+
+  private async createRound(game: IGame): Promise<string> {
+    const roundId = await roundService.create({
+      teamId: game.currentTeam,
+      hostId: await this.getHostIdFromTeam(game.currentTeam, game._id!),
+      chatId: await chatService.create(),
+      finishedAt: this.getFinishDate(new Date(), game.options.roundTime),
+      currentWord: await this.getRandomWord(game._id!)
+    } as IRoundCreateSchema)
+    return roundId;
+  }
+
+  async handleFinishedRound(id: string, round: IRound) {
+    const game = await this.getById(id);
+    let teamIndex = game.teams.findIndex(team => team.teamId === round.teamId);
+    game.teams[teamIndex].score += this.getScoreFromRound(round);
+    teamIndex = (teamIndex + 1) % game.teams.length;
+    game.currentTeam = game.teams[teamIndex].teamId;
+    await this.update(game);
+  }
+
+  private async getHostIdFromTeam(id: string, gameId: string): Promise<string> {
+    const team = await teamService.getById(id);
+    let rounds = await roundService.getAllByGameId(gameId);
+    rounds = rounds.filter(round => round.teamId = id);
```

Comment:
Maybe, there is a potential issue using the assignment operator and comparison:
```typescript
round => round.teamId === id
```

Reply:
what concrete issue do you mean?